README.md

```diff
@@ -1,46 +1,159 @@
-# Role Name
-
-A brief description of the role goes here.
+# sympa
+This is an Ansible role which sets up a sympa
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+Debian or Ubuntu
+A webserver that servers the sympa web app and a database for the web app.
 
 
 ## Role Variables
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
-```yml
+### Auth Variables
+
+`sympa_auth` list of auth methods used in order.
+each entry contains
+
+| Name      | Required/Default   | Description                                   |
+|:----------|:------------------:|:----------------------------------------------|
+| `name`    | :heavy_check_mark: | Name of the auth option like ldap, user_table |
+| `options` | :heavy_check_mark: | Dict of options for the auth method           |
+
+Example
+
+```yaml
+sympa_auth:
+  - name: ldap
+    options:
+    host: ldap.example.com:636
+      timeout: 20
+      suffix: yoursuffix
+      bind_dn: yourbinddn
+      bind_password: yourpw
+      use_tls: ldaps
+      ca_verify: none
+      get_dn_by_uid_filter: "(uid=[sender])"
+      email_attribute: mail
+      scope: sub
+      authentication_info_url: https://example.com
+  - name: user_table
+    options:
+      regexp: ".*"
 ```
 
+### Topic Variables
 
-## Dependencies
-
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-
-
-## Example Playbook
-
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+List of topics under the key `sympa_topics`
+Each list entry contains the following
 
+| Name    | Required/Default   | Description |
+|:--------|:------------------:|:------------|
+| `path`  | :heavy_check_mark: |     Path of the category        |
+| `title` | :heavy_check_mark: |     Title of the category        |
 
-### Playbook
+Example
 
-```yml
+```yaml
+sympa_topics:
+  - path: art
+    title: Art
+  - path: art/expressionism
+    title: Expressionism
 ```
 
+### Sympa Variables
```

Comment:
A Sympa documentation already exists, please link to it.

In general, I don't like the idea of maintaining a separate documentation of already documented technology, because we'd have to keep this up to date.

Reply:
sympa is not really developt and their documentation is spotty or outright wrong so I think a documentation that fits this role should be enough

Reply:
In my opinion a general _ansible way_ documentation is required, but please also add a link to the original documentation, this helps in case you need to lookup some detailed information about an option.

Reply:
> In my opinion a general _ansible way_ documentation is required

Can you please link to a resource explaining what an _ansible way_ documentation is?

If there are mistakes in the official documentation, then it's way more important to correct these individual mistakes here instead of trying to provide a generic documentation. Configuration management doesn't have the task to document existing projects but to deploy them. Therefore I think that your understanding that this is the _ansible way_ of doing it is wrong. Also, I don't think that the Ansible developers themselves encourge this kind of way since they don't have any examples or encouragement of re-documenting existing projects in the Ansible documentation.